content/accessibility/truncation.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Truncation
+---
+
+## Overview
+
+Truncation means shortening a text when it reaches a larger length than required, usually shortened with ellipses.
+
+There are several parts of our platform where we will have some text, which can probably contain focusable elements like links,
+and we won't have enough space to show it completely, so we wan't to truncate it. How do we truncate it in the most accessible way?
+
+The answer is unfortunately that we need to be a lot more explicit about how that truncation happens.
+We would need to identify a couple of use cases to understand the way that truncation can be done.
+
+## Implicit vs Explicit truncation
+
+### Implicit truncation
+
+Implicit truncation means that a certain text is truncated and the user has to take an action to make the text appear in full, although that action is not an explicit action using a control in the interface, meaning that the user can only expand by hovering or focusing it.
```

Comment:
```suggestion
Implicit truncation means that a certain text is truncated and the user has to take an implicit action such as hovering or focusing to make the text appear in full, rather than an explicit action such as using a control in the interface.
```